projects/qt5.cmake

```diff
@@ -171,7 +169,6 @@ function(build_qt5)
     SOURCE_DIR ${NULL_DIR} CONFIGURE_COMMAND "" BUILD_COMMAND ""
     INSTALL_COMMAND
       ${CMAKE_COMMAND} -E make_directory ${STAGE_DIR}/share/qt5 &&
-      ${CMAKE_COMMAND} -E copy ${SOURCE_DIR}/LGPL_EXCEPTION.txt ${STAGE_DIR}/share/qt5 &&
```

Comment:
Does this file no longer exist in 5.12.10?

Reply:
The file doesn't exist in 5.12.10, so this command causes the build to fail.